prebuilt_workers/prepare_prebuilt_workers/prepare_prebuilt_workers.go

```diff
@@ -0,0 +1,154 @@
+/*
+Copyright 2020 gRPC authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Configure is an executable that generates a defaults file for the manager.
+// It accepts a template file and replaces placeholders with data that may
+// change based on where the manager and container images will live and run.
+//
+// This tool uses Go's text/template package for templating, see
+// https://pkg.go.dev/text/template for a description of the syntax.
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+// Tests contains the values for fields that are accessible by
+// flags.
+type Tests struct {
+	version             string
+	preBuiltImagePrefix string
+	testTag             string
+	languagesToGitref   map[string]string
+	useCurrentTime      bool
+}
+
+type langFlags []string
+
+func (l *langFlags) String() string {
+	var result string
+	for _, lang := range *l {
+		result = result + " " + lang
+	}
+	return result
+}
+
+func (l *langFlags) Set(value string) error {
+	*l = append(*l, value)
+	return nil
+}
+
+var languagesSelected langFlags
+
+func main() {
+	var test Tests
+
+	// specify the PREBUILD_IMAGE_PREFIX
+	flag.StringVar(&test.preBuiltImagePrefix, "p", "", "image registry to push the image")
+
+	// specify the tag of pre built image
+	flag.StringVar(&test.testTag, "t", "", "tags for pre-built images, this unique tag to identify the images build and pushed in current test, if a tag is not provided, will user default tag as: build-initiator-date-time")
+
+	// specify the languages wish to run tests with
+	flag.Var(&languagesSelected, "l", "languages and its GITREF wish to run tests, example: cxx:master")
+
+	flag.Parse()
+
+	if test.preBuiltImagePrefix == "" {
```

Comment:
I don't think it is necessary to hardcode a default.

Reply:
Right, it make scene that users should have known where they wanted to push the images, otherwise the images should not be built. Will return a error message instead.